SimPPS/RPDigiProducer/plugins/RPGaussianTailNoiseAdder.cc

```diff
@@ -0,0 +1,53 @@
+#include "SimPPS/RPDigiProducer/interface/RPGaussianTailNoiseAdder.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "CLHEP/Random/RandGauss.h"
+#include <iostream>
+#include "TMath.h"
+#include "TRandom.h"
+
+using namespace std;
+
+RPGaussianTailNoiseAdder::RPGaussianTailNoiseAdder(int numStrips, 
+    double theNoiseInElectrons, double theStripThresholdInE, int verbosity)
+     : numStrips_(numStrips), theNoiseInElectrons(theNoiseInElectrons), 
+     theStripThresholdInE(theStripThresholdInE)
+{
+  verbosity_ = verbosity;
+  strips_above_threshold_prob_ = 
+      TMath::Erfc(theStripThresholdInE/sqrt(2.0)/theNoiseInElectrons)/2;
+}
+
+simRP::strip_charge_map RPGaussianTailNoiseAdder::addNoise(
+    const simRP::strip_charge_map &theSignal)
+{
+  the_strip_charge_map.clear();
```

Comment:
` the_strip_charge_map` seems to be used only in this function, in which case it would be better as a local variable.